OSSM-3251: Add cluster-wide deployment to Service Mesh docs (#58539)
openshift/openshift-docs: modules/ossm-add-project-using-label-selectors-console.adoc

// Module included in the following assemblies:
//
// * service_mesh/v2x/ossm-create-mesh.adoc

:_content-type: PROCEDURE
[id="ossm-adding-project-using-label-selectors-console_{context}"]
= Adding a project to the {SMProductShortName} using label selectors with the web console

You can use labels selectors to add a project to the {SMProductShortName} with the {product-title} web console.

> .Prerequisites

Reviewer: The list items need to use parallel grammatical contstructions -- all fragments or all complete sentences

Author: done

* An installed, verified {SMProductName} Operator.
* An existing `ServiceMeshMemberRoll` resource.

> * To perform this task, a user must have mesh admin privileges.

Reviewer: ```suggestion
* You are logged in as a user with mesh admin privileges.
```
- no need to say 'must'; it's implied by the fact that these are prereqs

Author: Done

.Procedure

. Log in to the {product-title} web console.

. Navigate to *Operators* -> *Installed Operators*.

> . Click the *Project* menu and choose the project where your `ServiceMeshMemberRoll` resource is deployed from the drop-down list. For example, `istio-system`.

Reviewer: ```suggestion
. Click the *Project* menu, and from the drop-down list, select the project where your `ServiceMeshMemberRoll` resource is deployed, such as *istio-system*.
```
- Per ISG: for drop-down lists, "se select to refer to selecting an item from a list." https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list
- Per ISG: use bold type for UI text and labels

Author: Prior peer reviews suggested breaking the "for example" part into a separate sentence, particularly when sentences are length. I'll keep the example as a separate clause, but thanks for the link about using select with drop-down menu. That's helpful for sure!

Done.

. Click the {SMProductName} Operator.

. Click the *Istio Service Mesh Member Roll* tab.

. Click *Create ServiceMeshMember Roll*.

. Accept the default name for the `ServiceMeshMemberRoll`.

> . In the *Labels* field, enter key value pairs to define the labels that identify which namespaces to include in the service mesh. If a project namespace has either label specified by the selectors then the project namespace is included in the service mesh. The project namespace does not need both labels to be included.

Reviewer: ```suggestion
. In the *Labels* field, enter key-value pairs to define the labels that identify which namespaces to include in the service mesh. If a project namespace has either label specified by the selectors, then the project namespace is included in the service mesh. You do not need to include both labels.
```

Author: Done
